saleor/graphql/invoice/mutations/invoice_request.py

```diff
@@ -75,7 +74,10 @@ def perform_mutation(  # type: ignore[override]
         cls.check_channel_permissions(info, [order.channel_id])
         cls.clean_order(order)
         manager = get_plugin_manager_promise(info.context).get()
-        if not is_event_active_for_any_plugin("invoice_request", manager.all_plugins):
+
+        if not manager.is_event_active_for_any_plugin(
+            "invoice_request", channel_slug=order.channel.slug
+        ):
```

Comment:
Yous need to cover the fixed scenario in test

Reply:
our tests already check for these scenarios, I just had to update the mocks 😅